src/components/DisambiguationPage.tsx

```diff
@@ -44,21 +44,12 @@ interface WordContentProps {
 }
 
 const WordContent: React.FC<WordContentProps> = (props) => {
-  const [isLoading, amount] = usePrefetchTerms(props.terms);
-  if (isLoading || props.parentLoading) return <Loader />;
```

Comment:
We can remove the `parentLoading: boolean;` prop from the interface because it is not used anywhere anymore